backend/battling/api/serializers.py

```diff
@@ -66,3 +79,80 @@ def create(self, validated_data):
 
         set_up_battle_teams_and_send_invite_email(instance)
         return instance
+
+
+class BattleTeamSerializer(serializers.ModelSerializer):
+    team = TeamSerializer(read_only=True)
+    pokemons = PokemonSerializer(many=True, read_only=True)
```

Comment:
Same thing here, this is a read-only field and should not be used on a write-only serializer